Documentation/operations/metrics.rst

```diff
@@ -286,146 +285,145 @@ report the eBPF memory usage directly managed by Cilium.
+``drop_bytes_total``                       ``reason``, ``direction``                          Enabled    Total dropped bytes
+``forward_count_total``                    ``direction``                                      Enabled    Total forwarded packets
+``forward_bytes_total``                    ``direction``                                      Enabled    Total forwarded bytes
+========================================== ================================================== ========== ========================================================
 Policy
 ~~~~~~
 
-========================================== ================================================== ========================================================
-Name                                       Labels                                             Description
-========================================== ================================================== ========================================================
-``policy``                                                                                    Number of policies currently loaded
-``policy_count``                                                                              Number of policies currently loaded (deprecated, use ``policy``)
-``policy_regeneration_total``                                                                 Total number of policies regenerated successfully
-``policy_regeneration_time_stats_seconds`` ``scope``                                          Policy regeneration time stats labeled by the scope
-``policy_max_revision``                                                                       Highest policy revision number in the agent
-``policy_import_errors_total``                                                                Number of times a policy import has failed
-``policy_endpoint_enforcement_status``                                                        Number of endpoints labeled by policy enforcement status
-========================================== ================================================== ========================================================
+========================================== ================================================== ========== ========================================================
+Name                                       Labels                                             Default    Description
+========================================== ================================================== ========== ========================================================
+``policy``                                                                                    Enabled    Number of policies currently loaded
+``policy_count``                                                                              Enabled    Number of policies currently loaded (deprecated, use ``policy``)
```

Comment:
It looks like `policy_count` was removed already, you could just drop this line.